### DISCUSSION TITLE
JavaScript Pig-Latin Generator

### DISCUSSION THREAD

**Discussion by h-enk** [thuliteio/doks#84] [Showcase]
![javascript-pig-latin-generator](https://user-images.githubusercontent.com/3902872/105137529-bbd84e00-5af3-11eb-94dc-486fedb3d818.png)

JPigLatin is a translation and speech synthesis application powered by the jPigLatin npm package.

👉 [jpiglatin.com](https://jpiglatin.com/)

**Comment by tcbutler320** [comment]
Awesome thanks for putting this up on here! About to launch another project with doks, love the theme!
